### DISCUSSION TITLE
Applying flags to segments

### DISCUSSION THREAD

**Discussion by ghost** [Flagsmith/flagsmith#1340] [Q&A] [answered]
Hi, we have a use case where users belong to separate organisations.

I can see that these organisations could be represented as segments in FlagSmith. 

<img width="412" alt="image" src="https://user-images.githubusercontent.com/447812/181391322-d13ff313-70ac-4ae1-8c3b-0d35b8662326.png">

What I would like to see though is a way to edit flags on a particular segment (organisation) without having to jump to the feature-flag view (or at least see the status of the feature flag before I edit it). 

Are there any plans to display the feature-flag status in this view or give the ability to update flags "inline" within this view?

Or is there a better way to manage this sort of thing?

**Comment by dabeeeenster** [comment]
Great idea! 

@kyle-ssg is this a lot of code?

**Reply by kyle-ssg** [reply]
Yeah, the original intention was to completely reuse the segment override table but we're quite limited in what the endpoint returns at the moment.

Let me take a look and see if I can somehow do this without any API adjustments, otherwise will write up a backend ticket.

**Reply by kyle-ssg** [reply]
Tracking under #1358

**Comment by kyle-ssg** [comment, marked as answer]
You can now edit segment overrides directly from this screen. We should also add a way to create new overrides for a feature from this screen. Tracking [here](https://github.com/Flagsmith/flagsmith/issues/1360).

**Reply by kyle-ssg** [reply]
https://user-images.githubusercontent.com/8608314/183994419-1030e41b-55cc-4ab7-b3df-197c0a4c2eab.mov
